src/main/java/xades4j/verification/SignatureSpecificVerificationOptions.java

```diff
@@ -61,6 +62,25 @@ String getBaseUri()
         return this.baseUriForRelativeReferences;
     }
 
+    public boolean checkKeyUsage()
```

Comment:
Can you make this method package private, please?

Reply:
Done in 955edde2315abbcc56d30bb74c0a5c51e282ce77.